### DISCUSSION TITLE
Prefix Zettelkasten creation date to file (YYYYMMDDHHmmss)

### DISCUSSION THREAD

**Discussion by carlosbaraza** [SilentVoid13/Templater#623] [Templates Showcase]
## What does it do?
Prefixes date ID in the format `YYYYMMDDHHmmss` to the filename and it updates all links. This is what [the Zettelkasten method describes](https://zettelkasten.de/posts/add-identity/).

## Template:
```javascript
<%*
    const id = tp.file.creation_date("YYYYMMDDHHmmss");
    await tp.file.rename(`${id} ${tp.file.title}`);
%>
```

## Why?
- I use this to add a date ordered identifier to my notes, and usually run this template after other templates.

**Comment by huyz** [comment]
You mean `mm` instead of `SS` right?

**Reply by carlosbaraza** [reply]
You are totally right, how dumb of a mistake that was. I don't create as many notes, so YYYYMMDDHH was all I needed really. It is corrected now to be `YYYYMMDDHHmmss`
